### PR TITLE
add instructions for running

### PR DESCRIPTION
**Pull request by Connoropolous**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,3 +2,13 @@
...
 Holochain is being rewritten in Rust as a set of crates that make up the Holochain Library. This means we need a container app to load Holochain and use its features. Using the cmd line wrapper [CLAP](https://github.com/kbknapp/clap-rs ) it was easy to parse the command line and pass those values to Holochain.
 As the library progresses we will add more exposed functions to the command line.
+
+## Quick How-to
+- have Rust installed
+
+In a terminal
+- clone this repository
+- change directories to this folder
+- run `cargo build`
```
Comment (sphinxc0re) [inline README.md]: If you actually want to install this tool:
```shell
cargo install -f --path .
```

Comment (Connoropolous) [reply README.md]: Oh, excellent!

**Comment by Connoropolous** [comment]
@sphinxc0re will you check now that I've changed it?

File: README.md [hunk 2]
```diff
@@ -2,3 +2,14 @@
...
+- run `cargo run` to see the CLI tools overview
+- run another command, such as `cargo run help`
```
Comment (sphinxc0re) [inline README.md]:
```shell
cargo run -- help
```

**Pull request merged**